### ISSUE TITLE
xontrib `cat` fails to exit when reading from sysfs

### ISSUE THREAD

**Issue by traverseda** [xonsh/xonsh#3199]
`/usr/bin/cat /sys/class/power_supply/BAT0/power_now ` works as expected
`cat /sys/class/power_supply/BAT0/power_now` never exits when using

```
$ cat --help    
This version of cat was written in Python for the xonsh project: http://xon.sh
```

**Comment by scopatz** [comment]
Thanks for reporting! I can read from certain things in `/sys/power`  but was having issues exiting

**Comment by laloch** [comment]
This seems to be fixed by #3200. @traverseda, can you please confirm?

**Comment by gforsyth** [comment]
Closing this out now -- if it's still an issue please ping here.
